tests/unit/lms/validation/_lti_launch_params_test.py

```diff
@@ -56,13 +57,13 @@ def test_LTIToolRedirect_raised_when_res_link_missing_with_return_url(
         pyramid_request.params.pop("resource_link_id")
         pyramid_request.params["launch_presentation_return_url"] = "http://example.com"
 
-        schema = LaunchParamsSchema(pyramid_request)
+        schema = BasicLTILaunchSchema(pyramid_request)
 
         with pytest.raises(LTIToolRedirect):
             schema.parse()
 
 
-class TestURLConfiguredLaunchParamsSchema:
+class TestURLConfiguredBasicLTILaunchSchema:
```

Comment:
I'm ambivalent about this as I haven't seen a great way to do it yet: a great way to avoid test duplication when you have a common private base class (or other common private helper of some sort) and you want to test it all without test code duplication